### PR TITLE
Attraction Page and Web Animation Development

### PR DESCRIPTION
**Pull request by aiwlulu**
丁育如 (Lulu Ding)

測試網址：
http://3.112.181.54:3000
http://3.112.181.54:3000/attraction/11

再麻煩老師了，謝謝～

### CODE REVIEW

**Review by cwpeng — APPROVED** [review]
Good. Well Done.

**Comment by cwpeng** [comment]
We have to merge pull request after review approval, not close it. If you don't get what these actions mean, try to figure out.

**Comment by aiwlulu** [comment]
已連同上週的一起補 merge，謝謝彭彭老師提醒！
之後會直接 merge & close

**Pull request merged**
